pkgs/desktops/plasma-5/ksysguard.nix

```diff
@@ -12,6 +13,6 @@ mkDerivation {
   nativeBuildInputs = [ extra-cmake-modules kdoctools ];
   buildInputs = [
     kconfig kcoreaddons kitemviews knewstuff kiconthemes libksysguard
-    kdelibs4support ki18n libcap libpcap lm_sensors
+    kdelibs4support ki18n libcap libpcap lm_sensors kdeFrameworks.networkmanager-qt libnl
```

Comment:
Generally we do not want to pass in one level lower than the package. Not sure if this is a common pattern in kde though.

Reply:
Yes, you MUST NOT pass in kdeFrameworks. Please refer to the Nixpkgs Manual for explanation.